Java/cp/MergeOverlappingArray.java

```diff
@@ -0,0 +1,78 @@
+package def;

```

Comment:
Add the description of the problem at the top.

Reply:
Done